static/js/hotkey.js

```diff
@@ -515,6 +515,12 @@ exports.process_hotkey = function (e, hotkey) {
         }
     }
 
+    // Prevents up and down navigation along the message list when lightbox is open.
+    if ((event_name === 'up_arrow' || event_name === 'down_arrow') && overlays.lightbox_open()) {
+        e.preventDefault();
```

Comment:
If we don't use the `preventDefault()`, we are able to move along the message list in the background in the upward direction  using the corresponding hotkey.